test/user.test.js

```diff
@@ -1763,6 +1763,54 @@ describe('User', function() {
     });
   });
 
+  describe('password reset with/without email verification', function() {
+    it('allows resetPassword by email if email verification is required and done',
+  function(done) {
```

Comment:
indentation is off, `function` should be indented with the same level as `it`.

```js
describe('...',
  it('...',
  function(done) {
  }):

  it('...',
  function(done) {
  });
```

Please fix all new tests.